daemon/main.c

```diff
@@ -106,12 +115,20 @@ void web_server_config_options(void) {
     if(!*web_x_frame_options) web_x_frame_options = NULL;
 
     web_allow_connections_from = simple_pattern_create(config_get(CONFIG_SECTION_WEB, "allow connections from", "localhost *"), NULL, SIMPLE_PATTERN_EXACT);
+    web_allow_connections_dns  = make_dns_decision("allow connections by dns","heuristic",web_allow_connections_from);
     web_allow_dashboard_from   = simple_pattern_create(config_get(CONFIG_SECTION_WEB, "allow dashboard from", "localhost *"), NULL, SIMPLE_PATTERN_EXACT);
+    web_allow_dashboard_dns  = make_dns_decision("allow dashboard by dns","heuristic",web_allow_dashboard_from);
     web_allow_badges_from      = simple_pattern_create(config_get(CONFIG_SECTION_WEB, "allow badges from", "*"), NULL, SIMPLE_PATTERN_EXACT);
+    web_allow_badges_dns  = make_dns_decision("allow badges by dns","heuristic",web_allow_badges_from);
     web_allow_registry_from    = simple_pattern_create(config_get(CONFIG_SECTION_REGISTRY, "allow from", "*"), NULL, SIMPLE_PATTERN_EXACT);
+    web_allow_registry_dns  = make_dns_decision("allow registry by dns","heuristic",web_allow_registry_from);
```

Comment:
Although the **allow registry by dns** is under the **registry** section, the **allow registry by dns** needs to be in the web section (``make_dns_decision`` uses the ``web`` section)

Reply:
Good point. Fixed.